apps/src/dance/lab2/views/DanceView.tsx

```diff
@@ -154,15 +154,15 @@ const DanceView: React.FunctionComponent<{
 
     return {
       analysis: [],
-      artist: 'A.I.', // TODO: user's name?
+      artist: '', // Unused
```

Comment:
See https://codedotorg.slack.com/archives/C099KT7NTHA/p1762212561144319 - this artist field is only used for the Song Names background effect I believe, and it seems that we're leaning towards removing it given the complexity around peaks metadata and the fact that it's already buggy on existing levels.